packages/mpc-tls/prover/src/handlers/proxy.rs

```diff
@@ -0,0 +1,75 @@
+use crate::{
+    proof::notarize::notarize_request,
+    proxy::{Header, ProxyRequest, BLACKLISTED_HEADERS},
+};
+use actix_web::{route, web, HttpRequest, HttpResponseBuilder, Responder};
+use hyper::body;
+use tracing::debug;
+use url::Url;
+
+#[route(
+    "/proxy",
+    method = "GET",
+    method = "POST",
+    method = "PUT",
+    method = "PATCH"
+)]
+pub async fn handle_notarization_request(payload: web::Payload, req: HttpRequest) -> impl Responder {
+    let t_proxy_url = req
+        .headers()
+        .get("T-PROXY-URL")
+        .expect("incomplete headers provided")
+        .to_str()
+        .unwrap();
+
+    let _t_store = req
+        .headers()
+        .get("T-STORE")
+        .map_or("", |value| value.to_str().unwrap_or_default());
+
+    let _t_redacted_parameters = req
```

Comment:
We're going to need this implemented.
Should determine the properties, within the verified proof response to redact.

Reply:
The PR for implementing this logic is included in the latest commit to this repo.